scripts/ci/run-integration-tests

```diff
@@ -110,9 +109,18 @@ function setup_plugins {
 # removes copied plugins files…
 function teardown_plugins {
   if [ -n "${PLUGINS_PATH}" ]; then
-    echo "==> Removing plugins folder…"
-    rm -r "${PROJECT_ROOT}/${TESTS_FOLDER}/${PLUGINS_FOLDER}/"
-    echo "===> done."
+    if [ ${CLEANUP_TESTS} -eq 0 ]; then
+      echo "==> Cleaning up ./${TESTS_FOLDER} folder"
+      rm -r ${PROJECT_ROOT}/${TESTS_FOLDER}
+      git checkout ${PROJECT_ROOT}/${TESTS_FOLDER}
```

Comment:
We rely on git anyways, I think what you really want to do here is a `git clean -dfx "${PROJECT_ROOT}/${TESTS_FOLDER}"` instead of rm and git checkout.

Reply:
good point, will change this if i touch the script again 👍